develop/dev-guide-gui-datagrip.md

```diff
@@ -0,0 +1,181 @@
+- Or A non-community edition [JetBrains](https://www.jetbrains.com/) IDE.
+
+Additionally, ensure you have the following:
+
+- A paid account that allows the use of DataGrip.
+- A TiDB cluster.
+
+<CustomContent platform="tidb">
+
+**If you don't have a TiDB cluster, you can create one as follows:**
+
+- (Recommended) Follow [Creating a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md) to create your own TiDB Cloud cluster.
+- Follow [Deploy a local test TiDB cluster](/quick-start-with-tidb.md#deploy-a-local-test-cluster) or [Deploy a production TiDB cluster](/production-deployment-using-tiup.md) to create a local cluster.
+
+</CustomContent>
+<CustomContent platform="tidb-cloud">
+
+**If you don't have a TiDB cluster, you can create one as follows:**
+
+- (Recommended) Follow [Creating a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md) to create your own TiDB Cloud cluster.
+- Follow [Deploy a local test TiDB cluster](https://docs.pingcap.com/tidb/stable/quick-start-with-tidb#deploy-a-local-test-cluster) or [Deploy a production TiDB cluster](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup) to create a local cluster.
+
+</CustomContent>
+
+## Connect to TiDB
+
+Connect to your TiDB cluster depending on the TiDB deployment option you've selected.
+
+<SimpleTab>
+<div label="TiDB Serverless">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+
+3. Ensure the configurations in the connection dialog match your operating environment.
+
+    - **Endpoint Type** is set to `Public`
+    - **Connect With** is set to `JDBC`
+    - **Operating System** matches your environment.
+
+4. Click **Create password** to create a random password.
+
+    > **Tip:**
+    >
+    > If you have created a password before, you can either use the original password or click **Reset password** to generate a new one.
+
+5. Create a project to manage your connections.
+
+    ![DataGrip create a project](/media/develop/datagrip-create-project.jpg)
+
+6. Click **+** in the upper-left corner of the **Database Explorer** panel, and select **Data Source** > **Other** > **TiDB**.
+
+    ![DataGrip select data source](/media/develop/datagrip-data-source-select.jpg)
+
+7. Copy the JDBC string from the TiDB Cloud connection dialog and replace the `<your_password>` placeholder with your actual password. Then, paste it into `URL`, and the remaining parameters will be auto-filled. The example result is as follows:
+
+    ![DataGrip URL paste](/media/develop/datagrip-url-paste.jpg)
+
+8. Click **Download** missing driver files. If you have used DataGrip before, you can skip this step.
+
+9. Click **Test Connection** to establish a connection to TiDB Serverless Tier cluster.
```

Comment:
Don't need to input CA path in DataGrip for tidb serverless? @Icemap

Reply:
Yep, the JDK uses the system's default CA path to link to the Serverless Tier.